qemu/tests/transfer_file_over_ipv6.py

```diff
@@ -26,25 +29,25 @@ def run(test, params, env):
     port = params.get("file_transfer_port")
     password = params.get("password")
     username = params.get("username")
-    tmp_dir = params.get("tmp_dir", "/tmp/")
+    if params.get("os_type") == "linux":
+        tmp_dir = params.get("tmp_dir", "/tmp/")
```

Comment:
suggest to put 'tmp_dir' in cfg file and set for different os type.

Reply:
@xiywang Thanks for review. Updated.